scripts/ci/setup.sh

```diff
@@ -6,6 +6,9 @@ ARGS+='this.dependencies_included = true;'
 # Process append to the version string if pre-release
 if [ "$TRAVIS_BRANCH" == "master" ]; then
     ARGS+="this.version += \"-alpha.$TRAVIS_BUILD_NUMBER\";"
+    # Run CI against latest master of submodules if this is master
+    git submodule update --remote api
```

Comment:
It's apparently how `--remote` works: https://git-scm.com/docs/git-submodule#git-submodule---remote